k8s/upgrades/0.9.0-1.0.0/jiva/verify_volume_upgrade.sh

```diff
@@ -0,0 +1,140 @@
+
+
+c_deploy_name=$(kubectl get deploy -n "$ns" \
+    -l openebs.io/persistent-volume="$pv",openebs.io/controller=jiva-controller \
+    -o jsonpath="{.items[*].metadata.name}" \
+)
+
+r_deploy_name=$(kubectl get deploy -n "$ns" \
+    -l openebs.io/persistent-volume="$pv",openebs.io/replica=jiva-replica \
+    -o jsonpath="{.items[*].metadata.name}" \
+)
+c_svc_name=$(kubectl get svc -n "$ns" \
+    -l openebs.io/persistent-volume="$pv" \
+    -o jsonpath="{.items[*].metadata.name}" \
+)
+
+#Fetch REPLICATIONFACTOR from deployment
+container_name=$(echo "$pv-ctrl-con")
+replication_factor=$(kubectl get deploy "$c_deploy_name" -n "$ns" \
+-o jsonpath="{.spec.template.spec.containers[?(@.name=='$container_name')].env[?(@.name=='REPLICATION_FACTOR')].value}")
+
+#Verifying the upgraded deployment versions
+controller_version=$(kubectl get deployment "$c_deploy_name" -n "$ns" \
+        -o jsonpath='{.metadata.labels.openebs\.io/version}')
+if [ "$controller_version" != "$target_upgrade_version" ]; then
+    echo "Failed to upgrade target deployment $c_deploy_name labels"
+    is_upgrade_failed=1
+fi
+replica_version=$(kubectl get deployment "$r_deploy_name" -n "$ns" \
+        -o jsonpath='{.metadata.labels.openebs\.io/version}')
+if [ "$replica_version" != "$target_upgrade_version" ]; then
+    echo "Failed to upgrade Replica deployment $r_deploy_name labels"
+    is_upgrade_failed=1
+fi
+
+#Verifying the upgraded service versions
+controller_svc_version=$(kubectl get svc "$c_svc_name" -n "$ns" \
+        -o jsonpath='{.metadata.labels.openebs\.io/version}')
+if [ "$controller_svc_version" != "$target_upgrade_version" ] ; then
+    echo "Failed to upgrade target service $c_svc_name labels"
+    is_upgrade_failed=1
+fi
+
+#Verifying the upgraded deployment images
+controller_images=$(kubectl get deployment "$c_deploy_name" -n "$ns" \
+        -o jsonpath='{range .spec.template.spec.containers[*]}{@.image}?{end}')
+for image in $(echo "$controller_images" | tr "?" " "); do
+    image_version=$(echo "$image" | cut -d ':' -f 2) 
+    if [ "$image_version" != "$target_upgrade_version" ] ; then
+        echo "Failed to upgrade controller deployment $c_deploy_name images"
+        is_upgrade_failed=1 
+    fi
+done
+
+replica_images=$(kubectl get deployment "$r_deploy_name" -n "$ns" \
+        -o jsonpath='{range .spec.template.spec.containers[*]}{@.image}?{end}')
+for image in $(echo "$replica_images" | tr "?" " "); do 
+    image_version=$(echo "$image" | cut -d ':' -f 2)
+    if [ "$image_version" != "$target_upgrade_version" ] ; then
+        echo "Failed to upgrade Replica deployment $r_deploy_name images"
+        is_upgrade_failed=1
+    fi
+done
+
+#Verifying running status of controller and replica pods
+running_ctrl_pod_count=$(kubectl get pods -n "$ns" \
+-l openebs.io/controller=jiva-controller,openebs.io/persistent-volume="$pv" \
+--no-headers | wc -l)
+if [ "$running_ctrl_pod_count" != 1 ]; then
```

Comment:
do we need '"' here?
can you run shellcheck and see whether it is doing any suggestions

Reply:
The `""` is suggested by shellcheck. 
```
Line 93:
if [ $running_ctrl_pod_count != 1 ]; then
     ^-- SC2086: Double quote to prevent globbing and word splitting.
```